### PR TITLE
Fixed issue #17378: Missing button: can not set permissions for survey group

### PR DESCRIPTION
**Pull request by gabrieljenik**
- Reinstated missing button.
- Fixed also on 'Add user', 'Delete user' and 'Add group users'
- Also added the green title bar that was recently introduced on Survey Groups but was missing on some pages (so the context was not clear)

### CODE REVIEW

**Comment by c-schmitz** [comment]
@gabrieljenik please resolve conflig.

File: application/controllers/SurveysGroupsPermissionController.php [hunk 1]
```diff
@@ -188,11 +188,16 @@ public function actionAddUser($id)
...
         );
         $aData['subview'] = 'addUserResult';
-        $aData['buttons'] = array(
-            'closebutton' => array(
-                'url' => App()->createUrl("SurveysGroupsPermission/index", array("id" => $id)),
-            ),
-        );
+
+        // Green Bar Page Title
+        $aData['pageTitle'] = 'Permission for group: ' . $model->title;
```
Comment (Shnoulle) [inline application/controllers/SurveysGroupsPermissionController.php:193]: gt ?

File: application/controllers/SurveysGroupsPermissionController.php [hunk 2]
```diff
@@ -239,11 +246,16 @@ public function actionAddUserGroup($id)
...
         );
         $aData['subview'] = 'addUserGroupResult';
-        $aData['buttons'] = array(
-            'closebutton' => array(
-                'url' => App()->createUrl("SurveysGroupsPermission/index", array("id" => $id)),
-            ),
-        );
+
+        // Green Bar Page Title
+        $aData['pageTitle'] = 'Permission for group: ' . $model->title;
```
Comment (Shnoulle) [inline application/controllers/SurveysGroupsPermissionController.php:251]: gt

File: application/controllers/SurveysGroupsPermissionController.php (hunk 1 above)
Comment (gabrieljenik) [reply application/controllers/SurveysGroupsPermissionController.php]: Solved. Thaks!

**Comment by gabrieljenik** [comment]
> @gabrieljenik please resolve conflict

Ready thanks!

**Pull request merged**